huaweicloud/config/hc_config.go

```diff
@@ -134,92 +135,49 @@ func getServiceEndpoint(c *Config, srv, region string) string {
 }
 
 // NewVpcClient is the VPC service client using huaweicloud-sdk-go-v3 package
-func NewVpcClient(c *Config, region string) (*vpc.VpcClient, error) {
-	credentials, err := buildAuthCredentials(c, region)
+func (c *Config) HcVpcV3Client(region string) (*vpcv3.VpcClient, error) {
+	hcClient, err := NewHcClient(c, region, "vpc", false)
 	if err != nil {
 		return nil, err
 	}
 
-	vpcEndpoint := getServiceEndpoint(c, "vpc", region)
-	if vpcEndpoint == "" {
-		return nil, fmt.Errorf("failed to get the endpoint of VPC service")
-	}
-
-	return vpc.NewVpcClient(
-		vpc.VpcClientBuilder().
-			WithEndpoint(vpcEndpoint).
-			WithCredential(*credentials).
-			WithHttpConfig(buildHTTPConfig(c)).
-			Build()), nil
+	return vpcv3.NewVpcClient(hcClient), nil
 }
 
 // NewTmsClient is the TMS service client using huaweicloud-sdk-go-v3 package
-func NewTmsClient(c *Config, region string) (*tms.TmsClient, error) {
-	credentials, err := buildGlobalAuthCredentials(c, region)
+func (c *Config) HcTmsV1Client(region string) (*tmsv1.TmsClient, error) {
```

Comment:
suggest to remove region from the pattern as TMS is a global service

Reply:
OK